.github/workflows/rubocop-powerhome.yml

```diff
@@ -0,0 +1,13 @@
+name: rubocop-powerhome
+
+on:
+  push:
+
+jobs:
+  ruby:
+    uses: ./.github/workflows/_ruby-package.yml
+    with:
+      package: ${{ github.workflow }}
+      ruby: '["2.7.4", "3.1.2"]'
+      rails: '["7.0.3.1","6.1.6.1","6.0.5.1","5.2.8.1"]'
```

Comment:
is there a reason we need to run these specific versions over `any` ?

Reply:
@that-jill `any` for `rubocop-cobra` was a way to make that collection not be empty and therefore get one test run with each ruby version. A workaround. `rubocop-cobra` doesn't need rails at all, that's basically what it means. `rubocop-powerhome` in the other hand does "need" rails (test with different rails versions).